detection-rules/link_login_or_captcha.yml

```diff
@@ -6,14 +6,12 @@ severity: "high"
 source: |
   type.inbound 
   and any(body.links, 
-      all([beta.linkanalysis(.)], (
-          .credphish.contains_login
-          or .credphish.contains_captcha
-      )
-      and (
-          .effective_url.domain.root_domain in $free_subdomain_hosts
```

Comment:
woops, i must've clobbered over these! we still want this check right?

Reply:
yessir